### PR TITLE
Merge Freya.Pipeline into Freya.Core #75

### PR DESCRIPTION
**Pull request by panesofglass**
Builds on the merge of `Freya.Integration` into `Freya.Pipeline`.

### CODE REVIEW

**Comment by panesofglass** [comment]
See [RELEASE_NOTES](https://github.com/panesofglass/freya/blob/merge/pipeline-to-core/RELEASE_NOTES.md)

File: src/Freya.Core/Integration.fs [hunk 1]
```diff
@@ -18,12 +18,12 @@
 //
 //----------------------------------------------------------------------------
 
-[<AutoOpen>]
-module Freya.Core.Integration
+module Freya.Integration
```
Comment (kolektiv) [inline src/Freya.Core/Integration.fs]: Do these need to live under separate spaces? Anything in Freya.Core should be Freya.Core.* I think, generally auto-opened as well. I'd be quite happy to lose the Freya.Pipeline and Freya.Integration namespaces and subsume them in to Freya.Core I think?

Comment (panesofglass) [inline src/Freya.Core/Integration.fs]: Sounds good to me. I was trying to make minimal changes. `Pipeline` and `Integration` are slightly different from the `Core` module types, so I don't think it's bad to leave them, but I also can't argue against using the same module/namespace for all.

Are you thinking `Freya.Core.Pipeline` and `Freya.Core.Integration` or just `Freya.Core` for everything?

Comment (kolektiv) [inline src/Freya.Core/Integration.fs]: I think those, but with [<AutoOpen>], so people only need to open
Freya.Core to get integration and pipelining available fully. I'll merge
locally in to my branch shortly and send an updated PR through soon!

Comment (panesofglass) [inline src/Freya.Core/Integration.fs]: Cool. I'll add `[<AutoOpen>]` shortly.

**Comment by panesofglass** [comment]
@kolektiv, ready for your review, sir!

**Comment by kolektiv** [comment]
I'm merging this in to my develop branch and tidying up the new router so I can get that in to freya-fs along with this today - it's a nice cleanup! We've lost two libs and things feel consistent :)

**Comment by panesofglass** [comment]
What do you say to merging this today?

**Comment by kolektiv** [comment]
I don't see why not!

**Comment by kolektiv** [comment]
Although, hang on!

**Comment by kolektiv** [comment]
This is superseded by #90, so if that gets merged, so does this :)

**Comment by panesofglass** [comment]
Is #90 ready?

**Comment by kolektiv** [comment]
Yes I think it's reasonably stable in terms of the features in there.

**Pull request merged**